modules/gcloud/gcloud.go

```diff
@@ -67,8 +70,26 @@ func (o Option) Customize(*testcontainers.GenericContainerRequest) error {
 
 // WithProjectID sets the project ID for the GCloud container.
 func WithProjectID(projectID string) Option {
-	return func(o *options) {
+	return func(o *options) error {
 		o.ProjectID = projectID
+		return nil
+	}
+}
+
+// WithDataYAML seeds the Bigquery project for the GCloud container with an io.Reader representing
+// the data yaml file, which is used to copy the file to the container, and then processed to seed
+// the Bigquery project.
+//
+// Other GCloud containers will ignore this option.
+// If this option is passed multiple times, the last added will be used.
+func WithDataYAML(r io.Reader) Option {
+	return func(o *options) error {
+		if o.bigQueryDataYaml != nil {
+			return errors.New("data yaml file already exists")
```

Comment:
suggestion: this isn't a file anymore so tweak the wording to match
```suggestion
			return errors.New("data yaml already set")
```